.github/workflows/ci-goreleaser-ebpf-profiler.yaml

```
@@ -0,0 +1,37 @@
+name: CI - eBPF Profiler - GoReleaser
+
+on:
+  merge_group:
+  push:
+    branches: [main]
+    paths:
+      - "distributions/otelcol-ebpf-profiler/**"
+      - "cmd/**"
+      - ".github/**"
+      - "scripts/**"
+      - "Makefile"
+      - "go.mod"
+      - "go.sum"
+  pull_request:
+    branches: [main]
+    paths:
+      - "distributions/otelcol-ebpf-profiler/**"
+      - "cmd/**"
+      - ".github/**"
+      - "scripts/**"
+      - "Makefile"
+      - "go.mod"
+      - "go.sum"
+
+jobs:
+  check-goreleaser:
+    name: CI - eBPF Profiler - GoReleaser
+    uses: ./.github/workflows/base-ci-goreleaser.yaml
+    with:
+      distribution: otelcol-ebpf-profiler
+      config_file: ebpf-profiler-config
```

why leave out the file extention here? i immediately confused this with a folder name. maybe it would be better for readability to include the extension.

```suggestion
      config_file: ebpf-profiler-config.yaml
```

(needs to be adjusted in the `base-ci-goreleaser.yaml` file as well

---

I wanted to avoid config be something else than `yaml`. But sure, I can change this.